js/stageCustomization.js

```diff
@@ -1,36 +0,0 @@
-;(async function replaceSiteTitle() {
```

Comment:
This is being handled here now: https://github.com/algorithmiaio/dev-center/blob/feature/FRONTEND-1813/server/customization.js